rmw_dds_common/package.xml

```diff
@@ -12,10 +12,12 @@
   <buildtool_depend>rosidl_cmake</buildtool_depend>
   <buildtool_depend>rmw_dds_common_generators</buildtool_depend>
 
+  <buildtool_export_depend>rosidl_cmake</buildtool_export_depend>
```

Comment:
Can you please describe why this package needs to export this dependency?

Reply:
The CMakeLists.txt is exporting the dependency `ament_export_dependency(rosidl_cmake)` https://github.com/ros2/rmw_dds_common/blob/master/rmw_dds_common/CMakeLists.txt#L25 but it was not declared in the package.xml

Reply:
Ok, why is the dependency exported from CMake?

Reply:
It's not needed.

I think that I initially didn't create `rmw_dds_common_generators`, which is already exporting a build tool dependency on `rosidl_cmake`, and then I forgot to delete this line from `rmw_dds_common`.

Reply:
Removed https://github.com/ros2/rmw_dds_common/pull/10/commits/c10bdd9617b91264437a6f286aa05f8a9689936b